docs/deployment/observability.md

```diff
@@ -2,16 +2,15 @@
 title: Setting up observability stack
 ---
 
-Script to set up observability stack (i.e. prometheus and grafana)
-- [hack/setup_observability.sh](hack/setup_observability.sh)
-This script requires few things
+A script exists to create the secrets and datasource used to connect Grafana with the Prometheus instance, required for observability ([hack/monitoring-template.sh](hack/monitoring-template.sh)).  
+This script requires few things:
 * [Github oauth](https://docs.github.com/en/developers/apps/building-oauth-apps/authorizing-oauth-apps) tokens for authentication of the components
 * [Github Cookie secret](https://oauth2-proxy.github.io/oauth2-proxy/docs/configuration/overview)
 * [oc](https://docs.openshift.com/container-platform/4.11/cli_reference/openshift_cli/getting-started-cli.html) binary installed and configured to have admin access to the cluster
 
-for running the `hack/setup_observability.sh` script
-1. Copy `hack/monitoring-template.env` to `hack/monitoring.env`
+for running the `hack/monitoring-template.sh` script
+1. Copy `hack/monitoring-template.sh` to `hack/monitoring.sh`
```

Comment:
Hello! There is no `monitoring-template.sh` in https://github.com/redhat-appstudio/infra-deployments/tree/main/hack, is that expected

Reply:
actually no, that's supposed to be `setup-monitoring.sh`, good catch :) fixed.